nifi-toolkit/nifi-toolkit-encrypt-config/src/main/groovy/org/apache/nifi/properties/ConfigEncryptionTool.groovy

```diff
@@ -64,8 +68,8 @@ class ConfigEncryptionTool {
     public String outputLoginIdentityProvidersPath
     public String authorizersPath
     public String outputAuthorizersPath
-    public String flowXmlPath
-    public String outputFlowXmlPath
+    public static flowXmlPath
+    public static outputFlowXmlPath
```

Comment:
Is there a reason these values are static?

Reply:
These were changed to static to allow static access in static main() for calling loadFlowXml(flowXmlPath). I changed this slightly to take the path as a parameter to allow easier testing for the loadFlowXml method. Not sure how successful it was though.